projects/views.py

```diff
@@ -350,6 +351,33 @@ def post(self, request, project_slug):
             project.authorized.add(selected_user)
             assign_perm("can_view_project", selected_user, project)
 
+            project_uri = f"{request.get_host()}/projects/{project.slug}"
+            email_body = f"""\
+Hi {selected_username},
+
+{request.user.username} added you to the project {project.name} on SciLifeLab Serve (https://serve.scilifelab.se)
+You can now view the project here: {project_uri}
+If you have any questions get in touch with us at serve@scilifelab.se
+
+Thank you for signing up for our service. We are excited to have you on board!
+
+Best regards,
+The Data Centre team
+"""
+
+            try:
+                # Notify user
+                send_mail(
+                    f"You've been added to {project.name}",
```

Comment:
I think it'd be good to add from which service does this email come from.

```suggestion
                    f"You've been added to {project.name} on SciLifeLab Serve",
```